server/serverAPI/controller/AuthenticationController.ts

```diff
@@ -17,12 +20,20 @@ import Token from "../model/token/Token";
 export default class AuthenticationController extends BaseUserController {
     private encryptor: Encryptor;
     private tokenCreator: TokenCreator<IIdentification>;
+    private emailAPI: IEmailAPI;
+
+    protected static verificationCodesMap: Map<string, { code: number, geneationTime: number }> = new Map();
+    protected verificationCodeLifetimeInMilliseconds = 15 *60 * 1000;
```

Comment:
Can you please make this 5 minutes? 15 is a long time to wait if you need another code